src/views/collection-view/style.module.css

```diff
@@ -0,0 +1,7 @@
+.newsletterSignupArea {
```

Comment:
Had to make a new css file since the other 2 in `collection-view` were scoped to their relevant components already, figured a general one for the entire component would be best